### ISSUE TITLE
Histogram Equalization using Rvision

### ISSUE THREAD

**Issue by matiasandina** [swarm-lab/Rvision#39]
This might be possible with the current status of the package, but I was trying to find a way to do histogram equalization.
This is implemented in python opencv ([see link](https://docs.opencv.org/master/d5/daf/tutorial_py_histogram_equalization.html))

So something like?

```
img <- Rvision::image("image.png")
img_eq <- Rvision::histEq(img)
```

**Comment by sjmgarnier** [comment]
@matiasandina I'm currently making big changes in the background of Rvision that I'm hoping to push this weekend. I can easily include `equalizeHist` (the name of the OpenCV C++ function) in that push. In the meantime, you can put your own histogram equalization function together using the imhist and LUT functions provided by Rvision.

**Comment by sjmgarnier** [comment]
Closed with 0.6.0 release.
